### PR TITLE
Branch update organization

### PR DESCRIPTION
**Pull request by tanshiyu1999**

### CODE REVIEW

**Review by McNaBry — APPROVED** [review]
Overall, looks good to me. The rids are saved nicely on the JSON and the applications runs fine overall. Further work is needed to make rids functional.

**Review by wxwern — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/person/Organization.java [hunk 1]
```diff
@@ -44,6 +51,13 @@ public Position getPosition() {
         return position;
     }
 
+    /**
+     * Returns an immutable tag set, which throws {@code UnsupportedOperationException}
+     * if modification is attempted.
+     */
+    public Set<Id> getRids() {
+        return Collections.unmodifiableSet(rids);
+    }
```
[the remark this answers is not in the document]
Comment (tanshiyu1999) [reply src/main/java/seedu/address/model/person/Organization.java]: The changes from `getRids()` to `getRecruiterId()` has been reflected.

**Review by wxwern — APPROVED** [review]
LGTM

Side note: the design choice for a new `--rid` flag to `Organization` may cause more confusion to the user due to the lack of auto-linkage with the reverse `--oid` link from `Recruiter`. Additionally, storing `rids` in JSON may significantly increase the chances of bugs. We might want to consider removing it for v1.2 and only include the functionality when it's ready instead.

**Review by McNaBry — APPROVED** [review]
Overall, looks good to me. In view of v1.2b this should be fine.

**Pull request merged**